code/modules/surgery/implant.dm

```diff
@@ -224,6 +224,24 @@
 		affected.hidden.update_icon()
 		affected.hidden = null
 
+	else if (((M_CLUMSY in user.mutations) || user.reagents.has_reagent(INCENSE_BANANA) || user.reagents.has_reagent(HONKSERUM)) && prob(20)) //I HATE DATUMS! I HATE DATUMS!
```

Comment:
```suggestion
	else if (((M_CLUMSY in user.mutations) || user.reagents.has_reagent(INCENSE_BANANA) || user.reagents.has_reagent(HONKSERUM)) && prob(20)) //I HATE DATUMS! I HATE DATUMS!
	else if(tool.clumsy_check(user) && prob(20))
```

Reply:
so i actually copy-pasted the clumsy_check into the fucking uhhhhhhhhhh thing instead of using clumsy_check because i assumed it didnt work inside a datum; youre telling me it didnt work because i needed tool in front of it? im going to shoot myself

Reply:
for some dumb reason clumsy_check() is an obj level proc instead of a mob level proc

it doesn't even ever get overwritten anywhere except in one case where it just returns 0

clumsy_check is a var too, if it's 0 it doesn't do clumsy_check() on some things